src/lib.rs

```diff
@@ -980,17 +980,24 @@ fn process_dom_node<'a, 'b, T: Write>(
                 expanded_name!(html "img") => {
                     let borrowed = attrs.borrow();
                     let mut title = None;
+                    let mut src = None;
                     for attr in borrowed.iter() {
                         if &attr.name.local == "alt" && !attr.value.is_empty() {
                             title = Some(&*attr.value);
+                        }
+                        if &attr.name.local == "src" && !attr.value.is_empty() {
+                            src = Some(&*attr.value);
+                        }
+                        if title.is_some() && src.is_some() {
                             break;
                         }
                     }
-                    if let Some(title) = title {
-                        Finished(RenderNode::new(Img(title.into())))
-                    } else {
-                        Nothing
-                    }
+                    return match (title, src) {
+                        (Some(title), Some(src)) => {
+                                Finished(RenderNode::new(Img(src.into(), title.into())))
+                        }
+                        _ => Nothing
+                    };
```

Comment:
I agree, that seems reasonable.  Perhaps in future it could be useful keeping the ones with `src` but not `alt`, but at the moment I don't think there's much use to it.